icechunk/src/repository.rs

```diff
@@ -1380,17 +1363,15 @@ mod tests {
         )
         .await?;
         let node = ds.get_node(&new_array_path).await;
-        assert_eq!(
+        assert!(matches!(
             node.ok(),
-            Some(NodeSnapshot {
```

Comment:
what's the difference here? They seem equivalent, and the new version feels less readable to me.

Reply:
yeah, very unfortunate ... we need to start writing some helpers for our tests. The issue is that I can no longer predict the node_id we'll get, so I cannot use `eq`.